public-html/404.php

```diff
@@ -0,0 +1,15 @@
+<?php
+http_response_code(404);
+require $_SERVER['DOCUMENT_ROOT'] . '/../vendor/autoload.php';
+
+use template\Page;
+
+$new = new Page;
+$content = <<<EOT
+<div class="main_content">
+    <h1 class="pagenotfound"> 
```

Comment:
its good practice to use classes like page-not-found

Reply:
mmmmm fine just a min

Reply:
Yeah fixed!